include/config/battle.h

```diff
@@ -187,6 +187,7 @@
 #define B_SMART_WILD_AI_FLAG        0     // If not 0, you can set this flag in a script to enable smart wild pokemon
 #define B_FLAG_NO_BAG_USE           0     // If this flag is set, the ability to use the bag in battle is disabled.
 #define B_FLAG_NO_CATCHING          0     // If this flag is set, the ability to catch wild Pokémon is disabled.
+#define B_FLAG_NO_RUNNING           0     // If this flag is set, the ability to escape from wild battles is disabled. Also makes Roar/Whirlwind fail.
```

Comment:
What about Teleport?

Reply:
and Roar

Reply:
> What about Teleport?

Doesn't teleport simply switch you out when used by the player?

Reply:
> and Roar / Whirlwind

Already included.

Reply:
Oh, I see. Teleport has a config. I'll adjust this tomorrow to cover that if necessary.

Reply:
You'd expect the flag to prevent any and all means of escape, so that'd be much appreciated. Thanks!